core/src/main/resources/lib/form/radioBlock.jelly

```diff
@@ -54,20 +54,22 @@ THE SOFTWARE.
 
   <st:adjunct includes="lib.form.radioBlock.radioBlock"/>
 
-	<tr class="radio-block-start row-group-start ${attrs.inline?'':'row-set-start'}" hasHelp="${attrs.help!=null}"><!-- this ID marks the beginning -->
-	  <td colspan="3">
+  <div class="tr radio-block-start row-group-start ${attrs.inline?'':'row-set-start'}" hasHelp="${attrs.help!=null}"><!-- this ID marks the beginning -->
+    <div colspan="3">
       <label>
-	      <input type="radio" name="${name}" value="${value}"
-             class="radio-block-control block-control" checked="${checked?'true':null}" />
+        <input type="radio" name="${name}" value="${value}"
+               class="radio-block-control block-control" checked="${checked?'true':null}" />
         ${title}
       </label>
-    </td>
+    </div>
     <f:helpLink url="${help}" featureName="${title}" />
-	</tr>
+  </div>
   <j:if test="${attrs.help!=null}">
     <f:helpArea />
   </j:if>
-	<d:invokeBody />
-	<!-- end marker -->
-	<tr class="${attrs.inline?'':'row-set-end'} radio-block-end row-group-end" style="display:none" />
+  <div class="container tr">
+    <d:invokeBody />
+  </div>
+  <!-- end marker -->
+  <div class="tr ${attrs.inline?'':'row-set-end'} radio-block-end row-group-end" style="display:none" />
```

Comment:
div tags are not self-closing, I think this should be `<div class="tr ${attrs.inline?'':'row-set-end'} radio-block-end row-group-end" style="display:none"></div>`